airflow_home/dags/modules/citytest_sf_appointments_report.py

```diff
@@ -0,0 +1,43 @@
+"""CityTestSF Appointment Report module"""
+import os
+import datetime
+from datetime import date
+from modules.acuity import Acuity
+from modules.citytest_sf_appointments import CityTestSFAppointments
+
+class CityTestSFAppointmentsReport(CityTestSFAppointments):
+    """CityTestSF Report for Appointments """
+
+    @staticmethod
+    def get_acuity_appointments(start_date=date.today(), end_date=date.today()):
+        """ get_acuity_appointments method """
+        acuity = Acuity()
+        appointments = []
+        default_params = {
+            'max': 2000
+        }
+        types = os.environ['CITYTEST_REPORT_APPT_TYPES'].split(',')
```

Comment:
do we have more than 1 appt type? or is this future proofing in case we get >1 kind of appt?

Reply:
Nope, not using it right now.  I had this in for when we were thinking if we should walk-up and drive up types and I guess also in case we want to use types to distinguish locations.